src/FavouriteRecipes.js

```diff
@@ -2,12 +2,11 @@ import React from "react";
 import Recipe from "./Recipe";
 
 const FavouriteRecipes = () => {
-  if (JSON.parse(localStorage.getItem("favourites")) === null){
-    localStorage.setItem('favourites', JSON.stringify([]));
-
+  if (JSON.parse(localStorage.getItem("favourites")) === null) {
+    localStorage.setItem("favourites", JSON.stringify([]));
   }
   const recipes = JSON.parse(localStorage.getItem("favourites"));
-  console.log(recipes)
+  console.log(recipes);
```

Comment:
Not related to this PR, but I noticed it now: all these console.log() lines should never make it to the production repo. They are great for debugging, but when submitting PR with code to be tested, they should all be gone. Could you create a card for that?